consensus/src/block_storage/sync_manager.rs

```diff
@@ -386,13 +386,19 @@ impl BlockStore {
 pub struct BlockRetriever {
     network: NetworkSender,
     preferred_peer: Author,
+    validator_addresses: Vec<AccountAddress>,
```

Comment:
not for this pr, but we probably should cache this vec somewhere if it's used a lot